### DISCUSSION TITLE
Workflow enhancements and suggestions

### DISCUSSION THREAD

**Discussion by Henri75** [apohl79/audiogridder#1205] [Help]
Hi,

Here are a few suggestions to make AG even better :

- oversampling support (for a specific plugin or chain) like in metaplugin which use Juce oversampling.
- plugin settings copy between AG instances
- copy/paste or even drag and drop plugins (with its settings) between AG instances. I remember someone speaking about that a while ago, and you seemed to be interested in that :)

Thanks for your amazing work

**Comment by apohl79** [comment]
Thank you. The last item is on the list already. Need to check on oversampling, as someone asked for this before.

**Comment by marcusvilano** [comment]
It would be awesome if AG have oversampling feature, great to use on mastering chain
